gravitee-node-vertx/src/main/java/io/gravitee/node/vertx/VertxFactory.java

```diff
@@ -160,7 +164,74 @@ public boolean isSingleton() {
         return true;
     }
 
-    private Set<String> loadLabels() {
+    private void loadMetricLabels() {
+        final Map<String, Set<Label>> includedLabelsByCategory = readConfiguredLabelsByCategory("include");
+
+        // Include labels takes precedence over defined labels.
```

Comment:
I really appreciate all those comments but I wonder if it's not preferable to have several little private function with their own intention. with time, it's easy to modify the code without modifying the comment.

Since your code is already well "split" by those comment it's easy to see dedicated methods
wdyt ?

Reply:
As discussed, after a second look, it seems already well split.